### ISSUE TITLE
Symlinke dir attached as volume works wrong way

### ISSUE THREAD

**Issue by whitecolor** [docker/for-win#222]
This is a problem can be seen with nodejs require:

if in container we have directory:
```
app/symlink_mapped (we got this by attaching volume `-v path/on/host/symlink:/app/symlink_mapped`)
app/node_modules/some-installed-module
```

Inside container we do:
1) cd symlink_mapped
2) node
3) require('some-installed-module') 

We will get the error:
```
Error: Can not find module 'some-installed-module'
```
if it wan not a symlink on the host, require in the container would work.

**Comment by withinboredom** [comment]
I usually mount `node_modules` as a named volume and have node modules installed by the container separate from the host, unless you expect windows executables to run in your container. Some node modules have native components.

**Comment by whitecolor** [comment]
I do with `node_modules` the same way. The problem is that nodejs modules that are located inside `app/symlink_mapped` can not `require` from modules that are in `node_modules` on the same level, though they should. I believe problem is not specific for `node`, but it was my case.

**Comment by withinboredom** [comment]
I've seen docker (on mac) randomly lose volumes and require a docker restart. It looks like an issue in node, exactly what you are seeing. Does this happen if you restart docker?

**Comment by whitecolor** [comment]
No, it just don't work. I had not experienced big problems with volumes, docker for windows is quite stable in my experience. Well it was one time I just had to reset to factory defaults. But this problem just is, did you dry to reproduce, or why?

**Comment by withinboredom** [comment]
Sorry, I just reread your issue, you have a different issue than I do :) I don't think docker supports symlinks in windows yet...

**Comment by whitecolor** [comment]
The problem is not in symlinks support. I just map a directory which are symlink on the dev machine, and in docker container it should be just like normal volume.

**Comment by whitecolor** [comment]
@dgageot I see the tag acknowledge so, I don't need to provide more info?

**Comment by friism** [comment]
@rneugeba is this related to, or a dupe of, https://github.com/docker/for-win/issues/137 ?

**Comment by rneugeba** [comment]
@whitecolor could you provide a simpler repro and example? The description above is a bit confusing for me.
thanks
